### ISSUE TITLE
Add CLI option to hide logo and other not super important text

### ISSUE THREAD

**Issue by mfronczyk** [seddonym/import-linter#364]
When import-linter is used as a guardrail by AI agents, every time it's invoked, the logo ascii art gets added to the context without a reason.

It'd be great to introduce some CLI argument to make the tool less verbose when there are no issues found, or at least remove the logo.

**Comment by seddonym** [comment]
More than happy to consider a PR that adds a `--no-logo` CLI option, i.e.

```
Options:
...
   --no-logo    Hide the logo.
```

**Comment by nf3lix** [comment]
I had the same thought and submitted a PR: https://github.com/seddonym/import-linter/pull/366
